BackofficeBundle/DependencyInjection/Compiler/TwigGlobalsCompilerPass.php

```diff
@@ -28,6 +28,10 @@ public function process(ContainerBuilder $container)
             $formResources[] = 'OpenOrchestraBackofficeBundle:Form:form_div_layout.html.twig';
             $container->setParameter('twig.form.resources', $formResources);
         }
+
+        $container->setParameter('jms_serializer.handlers.datetime.default_format', \DateTime::ISO8601);
```

Comment:
What is the relationship with twig. And where this parameter is used by DateHandler of  jms serialize ? because is the definition of service this parameter isn't used